README.md

```diff
@@ -1,69 +1,273 @@
 # reply-urls-operator
-// TODO(user): Add simple overview of use/purpose
+A k8s Operator that keeps Ingress hosts in sync with the Redirect URLs associated with an Azure App Registration.
+
+## Table of Contents
+**[Description](#Description)**<br>
+**[How the Operator works](#How-the-Operator-works)**<br>
+**[Deploying the Operator to a cluster](#Deploying-the-Operator-to-a-cluster)**<br>
+**[Test out the operator locally](#Test-out-the-operator-locally)**<br>
+**[GitHub Workflows](#GitHub-Workflows)**<br>
+**[Modifying the API definitions](#Modifying-the-API-definitions)**<br>
+**[License](#License)**<br>
```

Comment:
GitHub automatically adds a table of contents 

no need to have a manual one

![image](https://user-images.githubusercontent.com/21194782/191942238-f3e32956-5300-401b-b24c-302c783a9f83.png)

Reply:
```suggestion
```